yaml-cpp.sh

```
@@ -1,12 +1,13 @@
 package: yaml-cpp
 version: "%(tag_basename)s"
-tag: 0.8.0
+tag: yaml-cpp-0.9.0
 source: https://github.com/jbeder/yaml-cpp
 build_requires:
   - CMake
   - alibuild-recipe-tools
 prefer_system: (?!slc5)
 prefer_system_check: |
```

Shellcheck (0.11.0)

[error] Expected a { to open the function definition.

(SC1064)